DynamicProgramming/placing parenthesis

```diff
@@ -0,0 +1,27 @@
+import java.util.Scanner;
+
+public class PlacingParentheses {
```

Comment:
Please use proper file name for this file - PlacingParentheses.java

Reply:
sir i have commit 77116222b67ed4c5600586c93b58e2e5e7fc68e9 changes that you suggest